CHANGES.txt

```diff
@@ -5,6 +5,8 @@ Changes for Crate Data JDBC Client
 Unreleased
 ==========
 
+ - Implemented ability to set JDBC client properties via the connection string.
```

Comment:
suggestion: It is now possible to set connection properties via the connection string